qa/tasks/cephfs/test_scrub_checks.py

```diff
@@ -443,6 +443,56 @@ def test_scrub_split_dirfrags(self):
             timeout=30
         )
 
+    def test_scrub_backtrace_divergent_newer(self):
+        """
+        On-disk backtrace is divergent or newer
+        """
+
+        test_path2 = "testdir1/testdir2"
+        test_path3 = f"{test_path2}/testdir3"
+        test_path4 = f"{test_path3}/testdir4"
+        abs_test_path4 = f"/{test_path4}"
+        split_size = 100
+        merge_size = 50
+        replica_threshold = 1
+
+        self.config_set('mds', 'mds_bal_split_size', split_size)
+        self.config_set('mds', 'mds_bal_merge_size', merge_size)
+        self.config_set('mds', 'mds_bal_replicate_threshold', replica_threshold)
+
+        self.mount_a.run_shell(["mkdir", "-p", test_path4])
+        dir_ino=self.mount_a.path_to_ino(test_path4)
+
+        self.assertEqual(len(self.get_dsplits(dir_ino)), 0)
+        for i in range(8 * split_size):
+            self.mount_a.run_shell(["touch", f"{test_path4}/file_{i}"])
```

Comment:
This will be unnecessary slow, move to bash `for` loop:

```
f"cd {path} && for i in `seq 1 {8*split_size}`; do touch file_$i; done"
```

Reply:
Looks nice.